settings.gradle

```diff
@@ -15,9 +15,13 @@
  */
 
 rootProject.name = 'autodispose-root'
+include ':android:autodispose-android'
+include ':android:autodispose-android-kotlin'
+include ':android:autodispose-android-archcomponents'
+include ':android:autodispose-android-archcomponents-kotlin'
+include ':android:autodispose-android-archcomponents-test'
+include ':android:autodispose-android-archcomponents-test-kotlin'
```

Comment:
Well, I have some concerns here. Looks like we have few dimensions (like lifecycle events provider, platform, language), and implementing or updating new thing for any of it leads to rewriting of N sublibraries. Maybe you have an idea how to resolve it, or we can discuss offline and try to find a solution.

Reply:
What do you mean by rewriting?

Reply:
Like, imagine, you want to update autodispose-kotlin. It'll (possibly) lead to updates in autodispose-android-archcomponents-kotlin and autodispose-android-archcomponents-test-kotlin (actually, any artifact mentions kotlin).The same condition will stay for, say, update in archcomponents. If we have it somewhat decoupled (I don't have solution for this, it's only an brainstorm-ish idea!), then we can leverage modularity better. Also, it'll be better for library consumers to have more granular control over dependencies.

Reply:
Ah, no the plan is for them to always be in sync even if some artifacts have no changes